app/src/pages/playground/fetchPlaygroundPrompt.ts

```diff
@@ -6,6 +6,7 @@ import {
 } from "@phoenix/constants/generativeConstants";
```

Comment:
It's about time that I move and rename this file to like, `src/utils/promptPlaygroundConversionUtils/utils.ts`

It gets imported across pages now.

I don't want to do it in this PR though and obfuscate my real changes